sirius-map-ios/SiriusMap/Search/SearchNavigationController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SearchNavigationController.swift
+//  SiriusMap
+//
+//  Created by Egor on 21.03.2024.
+//
+
+import UIKit
+
+extension String {
+    static let title = "Куда"
+    static let cancel = "Отменить"
+}
+
+final class SearchNavigationController: UINavigationController {
+    
+    // MARK: - Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let viewController = makeSearchViewController()
+        addCancelButton(to: viewController)
+        setViewControllers([viewController], animated: false)
+    }
+    
+    // MARK: - Methods
+    
+    private func makeSearchViewController() -> UIViewController {
+        let searchViewController = SearchTableViewController(items: MapItem.sampleData.map { $0.name })
+        searchViewController.delegate = self
+        searchViewController.title = .title
+        return searchViewController
+    }
+    
+    private func addCancelButton(to viewController: UIViewController) {
+        viewController.navigationItem.leftBarButtonItem = UIBarButtonItem(
+            title: .cancel,
+            primaryAction: UIAction { _ in
```

Comment:
```suggestion
            primaryAction: UIAction { [weak viewController] _ in
```
иначе получим утечку